src/main/kotlin/org/kamiblue/client/command/commands/ModifiedCommand.kt

```diff
@@ -0,0 +1,37 @@
+package org.kamiblue.client.command.commands
+
+import net.minecraft.util.text.TextComponentString
+import net.minecraft.util.text.event.ClickEvent
+import net.minecraft.util.text.event.HoverEvent
+import org.kamiblue.client.command.ClientCommand
+import org.kamiblue.client.module.modules.client.CommandConfig
+import org.kamiblue.client.util.text.MessageSendHelper.sendChatMessage
+
+object ModifiedCommand: ClientCommand(
+    name = "modified",
+    description = "View modified settings in a module"
+) {
+    init {
+        module("module") {
+            execute("List modified settings") {
+                var anyChanged = false
+
+                for (setting in it.value.settingList) {
+                    if (!setting.isModified) continue
+                    anyChanged = true
+
+                    val component = TextComponentString("${setting.name} has been changed to ${setting.value}")
+                    // horrible, however this is mojang code that we are working on.
+                    component.style.clickEvent = ClickEvent(ClickEvent.Action.RUN_COMMAND, "${CommandConfig.prefix.value}set ${it.value.name} ${setting.name.replace(" ", "")} ${setting.defaultValue}")
```

Comment:
Please use CommandManager.prefix instead of manually accessing the prefix

Reply:
Sorry, didn't know that it was stored there.